### PR TITLE
Rotating hero images

### PR DESCRIPTION
**Pull request by andreas-groos**
Cycles through an array of predefined bg-images. Currently it's just some lorempixel placeholders with a tech-theme. It assumes a 5:1 ratio in the CSS to scale but that could be changed. If you have actual images to add (preferably tech or Nevada County themed) just add them.
At the moment it changes every 10 seconds, probably should be set to something slower but I made it relatively quick so the effect could be easier observed for testing.... Just change the secs variable in ui.js

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -50,11 +46,11 @@ body {
       font-family: expletus, serif;
       font-size: 4vw;
       text-shadow: 0px 4px 4px rgba(0, 0, 0, 0.7);
-      margin: .5vw 0; }
```
Comment (djdefi) [inline main.css]: I believe this was changed in #13 to use `vw` values.

cc: @dennis-wolfers

File: main.css [hunk 2]
```diff
@@ -50,11 +46,11 @@ body {
...
+      margin: 10px 0; }
     .hero .overlay h3 {
       color: white;
       font-size: 2vw;
-      margin: .3vw 0; }
```
Comment (djdefi) [inline main.css]: Also here, this was changed in #13

**Review by djdefi — COMMENTED** [review]
Couple comments about a change that was introduced in #13 -- Personally I don't have an opinion on one way vs the other.

File: main.css (hunk 1 above)
Comment (andreas-groos) [reply main.css]: Yeah, my bad, not sure how that slipped back given that I thought I pulled the newest version before adding new code. I'm away from my computer, so won't be able to change things till Tuesday.

Comment (andreas-groos) [reply main.css]: Yeah, my bad, not sure how that slipped back given that I thought I pulled the newest version before adding new code. I'm away from my computer, so won't be able to change things till Tuesday.

**Comment by andreas-groos** [comment]
I went back to vw units in .hero .overlay h1 and .hero .overlay h3. My mistake to reverting to an old version by accident, I thought I had pulled the newest version from github. 
Also added actual landscape images of Nevada County and got rid of the tech-themed placeholder images.
If you got more images you can add them to the img folder and add the filename in bgImages in ui.js

**Review by djdefi — APPROVED** [review]

**Pull request merged**